setup.py

```diff
@@ -42,12 +42,12 @@ def get_install_requires():
     name='openwisp-notifications',
     version=get_version(),
     license='GPL3',
-    author='Gagan Deep',
```

Comment:
I think this `author` and `author_email` should be maintained since it is your work. :smile:

Reply:
:+1: